src/graphics/vertex_buffer.rs

```diff
@@ -224,16 +254,6 @@ impl VertexBuffer {
     }
 }
 
-impl Clone for VertexBuffer {
```

Comment:
I don't see the need to remove the original panicking Clone implementation for `VertexBuffer`.
It's there as an option if you don't care about handling a clone failure.